packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -18,9 +18,13 @@ export default ({ api }: { api: APIType }) => {
     }
   }
 
+  const filterTerminatedCards = (cards) => cards.filter((card) => card.status !== 'TERMINATED')
```

Comment:
how many statuses are there going to be?  we should create an enum sooner rather than later. dont want to litter code with same strings everywhere

Reply:
I understand, but this has close to nothing defined. No it will be only be used here, in the moment I need to use this string in other place I will create a enum an avoid duplication